index.html

```diff
@@ -2,36 +2,62 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<!-- Link your CSS stylesheet -->
+<link rel="stylesheet" href="index.css">
     
 </head>
 <body>
-        Plan your Visit
-        Learn about the Park
-        Get Involved
-   
-      
-      "header_img.jpg"
-      Yosemite Park Guide
-      Come explore California’s beautiful wilderness.
-      Always open. Reservations are not available or required to enter the park.
-      
-      Not just a great valley, but a shrine to human foresight, the strength of granite, the power of glaciers, the persistence of life, and the tranquility of the High Sierra.
-      
-      First protected in 1864, Yosemite National Park is best known for its waterfalls, but within its nearly 1,200 square miles, you can find deep valleys, grand meadows, ancient giant sequoias, a vast wilderness area, and much more.
-      
-      "https://image.flaticon.com/icons/svg/2979/2979348.svg"
-      Fire is a natural and essential part of Yosemite. We manage fire carefully and study how it interacts with the park’s ecosystems. This blog provides updates about fires in Yosemite. Check for current fire restrictions.
-      
-      "https://image.flaticon.com/icons/svg/820/820374.svg"
-      We strongly recommend that you make reservations for lodging, camping, and backpacking. You can pay the park entrance fee upon arrival (there's no need to pay it in advance).
-    
-      "footer_img.png"
-      
-      
-        Plan your Visit
-        Learn about the Park
-        Get Involved
+    <header>
+    <nav class="topmenu">
+  <a href="plan" class="one">Plan Your<br>Visit</a>
+  <a href="about" class="two">Learn About<br>The Park</a>
+  <a href="#" class="three">Get<br>Involved</a>
+    </nav>
+        <div class="headpiece">
+          <div class="left">
+            <div class="img">
+      <img src="header_img.jpg">
+      alt="mountain top at sunset"
```

Comment:
The alt tag should live within your `img` element. `<img src="header_img.jpg"  alt="mountain top at sunset">`